### PR TITLE
tmuxPlugins.session-wizard: add at 1.2.0

### PR DESCRIPTION
**Pull request by mandos**
- maintainers: add mandos
- tmuxPlugins.session-wizard: init at 1.2.0

## Description of changes

Tmux plugin to manage session based on visited directories, uses fzf and zoxide.
https://github.com/27medkamal/tmux-session-wizard

### CODE REVIEW

File: pkgs/misc/tmux-plugins/default.nix [hunk 1]
```diff
@@ -548,6 +548,39 @@ in rec {
     '';
   };
 
+  session-wizard = mkTmuxPlugin rec {
+    pluginName = "session-wizard";
+    # rtpFilePath = "session-wizard.tmux";
```
Comment (ambroisie) [inline pkgs/misc/tmux-plugins/default.nix:553]: Dead code.

**Review by ambroisie — CHANGES_REQUESTED** [review]
Tiny nit, otherwise LGTM.

File: pkgs/misc/tmux-plugins/default.nix (hunk 1 above)
Comment (mandos) [reply pkgs/misc/tmux-plugins/default.nix]: Sorry, it's not dead code, it's my mistake. I don't know what is normal flow so I amended it and push forced.

Comment (ambroisie) [reply pkgs/misc/tmux-plugins/default.nix]: Amending and pushing is fine, don't worry.

**Review by ambroisie — APPROVED** [review]

**Review by Janik-Haag — APPROVED** [review]

**Pull request merged**